packages/suite/src/views/wallet/sign-verify/index.tsx

```diff
@@ -65,17 +65,17 @@ const StyledSelectBar = styled(SelectBar)`
 
     @media (min-width: ${variables.SCREEN_SIZE.SM}) {
         width: 320px;
-        margin: 0 0 0 20px;
+        margin: 6px 0 0 20px;
     }
 
     @media (min-width: ${variables.SCREEN_SIZE.MD}) and (max-width: ${variables.SCREEN_SIZE.LG}) {
         width: 100%;
-        margin: 12px 0 20px;
+        margin: 6px 0 20px;
```

Comment:
it looked less crowded with `12px` and the distance between the two inputs above is `12px` too. 

<img width="545" alt="image" src="https://github.com/trezor/trezor-suite/assets/45338719/f6ac3242-c15c-4357-81fb-d4ddf0c26c99">